bin/bundle

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
```

Comment:
Whats this file about?

Reply:
This is an executable file which allows us to run the bundler-cli that has commands like `bundle ...`

Reply:
Note that this file & command is the same as yarn.